### ISSUE TITLE
AnimatePolygon does not close the shape that is defined by spec

### ISSUE THREAD

**Issue by citrusvanilla** [salabim/salabim#37]
AnimatePolygon should close an unclosed polygon for drawing purposes.

```
sim.AnimatePolygon(
    spec=[
        *(-75.0, -75.0), #sw
        *(75.0, -75.0), #nw
        *(75.0, 75.0), #ne
        *(-75.0, 75.0) #se
    ],
    linewidth=5,
    ...
)
```

<img width="199" alt="image" src="https://user-images.githubusercontent.com/7883733/206302780-3994d0be-7eed-4696-bd9c-851b7f663b19.png">

**Comment by citrusvanilla** [comment]
Suggested fix submitted in PR #38

**Comment by salabim** [comment]
Thanks. This will be fixed in the next release.
